config/default/kustomization.yaml

```diff
@@ -12,21 +12,20 @@ namePrefix: etcd-druid-
 #commonLabels:
 #  someName: someValue
 
-bases:
-- ../crd
-- ../rbac
-- ../manager
+resources:
```

Comment:
Each entry in this list must resolve to an existing resource definition in YAML.  These are the resource files that kustomize reads, modifies and emits as a YAML string, with resources separated by document markers ("---"). For bases, thats not the case.

Checkout: https://kubectl.docs.kubernetes.io/pages/examples/kustomize.html

Reply:
Bases is deprecated. https://kubernetes-sigs.github.io/kustomize/api-reference/kustomization/bases/